Reduce examples

kristinawlai opened the pull request:
Reduced examples by filtering curve params to HylE_IgA and HylE_IgG and limiting it to 1000 iterations. Also removed ggfortify from documentation as it's no longer in use.

kristinawlai committed:
reduce curve params to HylE_IgA and HylE_IgG and limit to 1000 iterations

kristinawlai committed:
remove ggfortify from documentation

d-morrison commented on R/autoplot.curve_params.R:
```diff
@@ -13,10 +13,13 @@
-#'   load_curve_params() %>%
+#' curve = load_curve_params("https://osf.io/download/rtw5k/") %>%
+#' filter(antigen_iso == c("HlyE_IgA", "HlyE_IgG")) %>%
+#'   slice(1:1000)  %>% #reduce dataset for the purposes of this example
```
- [x] please add `.by = antigen_iso` to the `slice()` call, to avoid the possibility of getting only one antigen

d-morrison replied:
(it works currently, because the input RDS file is ordered by `iter` before `antigen_iso`, but that might change)

d-morrison reviewed (changes requested):
see comment; make sure that `slice()` groups by `antigen_iso`

kristinawlai committed:
Reduce curve params to 100 iterations and remove donttest.

kristinawlai committed:
Merge branch 'main' into reduce-examples

kristinawlai committed:
Increment version number to 1.0.1

kristinawlai committed:
Add resubmission section

kristinawlai committed:
Merge branch 'reduce-examples' of https://github.com/UCD-SERG/serocalculator into reduce-examples

d-morrison commented on R/autoplot.seroincidence.by.R:
```diff
@@ -7,16 +7,17 @@
-#' load_curve_params()
+#' curve = load_curve_params("https://osf.io/download/rtw5k/") %>%
+#' filter(antigen_iso == c("HlyE_IgA", "HlyE_IgG")) %>%
+#'   slice(1:100, .by=antigen_iso) #reduce dataset for the purposes of this example
```
- [x] Please format examples according to the style guide: https://style.tidyverse.org/syntax.html#infix-operators

d-morrison commented on R/autoplot.summary.seroincidence.by.R:
```diff
@@ -10,12 +10,16 @@
-#' curve = load_curve_params("https://osf.io/download/rtw5k/" )
+#' curve = load_curve_params("https://osf.io/download/rtw5k/") %>%
+#' filter(antigen_iso == c("HlyE_IgA", "HlyE_IgG")) %>%
+#'   slice(1:100, .by=antigen_iso) #reduce dataset for the purposes of this example
```
- [x] again see https://style.tidyverse.org/syntax.html#infix-operators, and also [the section on comments](https://style.tidyverse.org/syntax.html#infix-operators:~:text=Each%20line%20of%20a%20comment%20should%20begin%20with%20the%20comment%20symbol%20and%20a%20single%20space%3A%20%23) (I won't flag remaining instances, but do this everywhere)

d-morrison reviewed (changes requested):
I noticed some minor style issues in the examples; see comments. 

I think CRAN will accept the examples now; after making the style changes, please try resubmitting!

kristinawlai committed:
Update examples accordin to tidyverse style guide

kristinawlai committed:
Update examples accordin to tidyverse style guide

kristinawlai committed:
More updates to style

Merge branch 'reduce-examples' of https://github.com/UCD-SERG/serocalculator into reduce-examples

# Conflicts:
#	R/autoplot.curve_params.R
#	R/autoplot.seroincidence.R
#	R/autoplot.seroincidence.by.R
#	R/autoplot.summary.seroincidence.by.R
#	R/check_pop_data.R
#	R/est.incidence.R
#	R/est.incidence.by.R
#	R/getAdditionalData.R
#	R/graph.loglik.R
#	R/llik.R
#	R/summary.seroincidence.R
#	R/summary.seroincidence.by.R

kristinawlai committed:
Merge branch 'main' into reduce-examples

d-morrison commented on R/autoplot.curve_params.R:
```diff
@@ -14,8 +14,8 @@
 #' curve = load_curve_params("https://osf.io/download/rtw5k/") %>%
-#' filter(antigen_iso == c("HlyE_IgA", "HlyE_IgG")) %>%
-#'   slice(1:100, .by=antigen_iso)  %>% #reduce dataset for the purposes of this example
+#'   filter(antigen_iso %in% c("HlyE_IgA", "HlyE_IgG")) %>%
```
good catch!

d-morrison commented on R/autoplot.pop_data.R:
```diff
@@ -13,14 +13,13 @@
-#' xs_data %>% autoplot(strata = "Country", type = "density")
-#' xs_data %>% autoplot(strata = "Country", type = "age-scatter")
+#' xs_data %>% autoplot(strata <- "Country", type = "density")
+#' xs_data %>% autoplot(strata <- "Country", type = "age-scatter")
```
using `<-` for function inputs doesn't work; it's only passing the GitHub checks because you haven't run `document()` since making these changes, and the examples are pulled from the .Rd files.

d-morrison commented on R/autoplot.seroincidence.R:
```diff
@@ -12,25 +12,25 @@
-#' noise = load_noise_params("https://osf.io/download//hqy4v/")
+#' curve <- load_curve_params("https://osf.io/download/rtw5k/") %>%
+#'   filter(antigen_iso %in% c("HlyE_IgA", "HlyE_IgG")) %>%
+#'   slice(1:100, .by=antigen_iso) # Reduce dataset for the purposes of this example
```
still missing spaces around `=`

d-morrison commented on R/autoplot.summary.seroincidence.by.R:
```diff
@@ -15,20 +15,22 @@
-#' antigen_isos = c("HlyE_IgG", "HlyE_IgA"),
-#' #num_cores = 8 #Allow for parallel processing to decrease run time
+#' curve <- load_curve_params("https://osf.io/download/rtw5k/") %>%
+#' filter(antigen_iso %in% c("HlyE_IgA", "HlyE_IgG")) %>%
```
are you doing the indenting by hand, or did you find a way to automate it? Either way, line 22 needs an indent.

kristinawlai replied:
Not fancy enough to automate it, so I did it by hand haha.

d-morrison commented on R/graph.decay.curves.R:
```diff
@@ -19,10 +19,10 @@
-#' load_curve_params() %>%
-#' dplyr::filter(antigen_iso == "HlyE_IgG") %>%
-#' serocalculator:::plot_curve_params_one_ab()
+#'   "https://osf.io/download/rtw5k/" %>%
```
line 22 doesn't need an indent, I think; check the formatted help file and make sure it looks correct?

Currently, I'm getting this:

![image](https://github.com/UCD-SERG/serocalculator/assets/2474437/3acd5231-793d-4202-a6f5-470f8784c58b)

The second line is the start of a new command, so it shouldn't be indented

kristinawlai replied:
Removed indent and updated this to put the URL inside of `load_curve_params()`

d-morrison reviewed (changes requested):
Thanks for the thorough edits! I flagged a few remaining issues; most importantly, remember to run `document()` as the last step of editing the documentation.

kristinawlai committed:
More style updates

kristinawlai committed:
Merge branch 'reduce-examples' of https://github.com/UCD-SERG/serocalculator into reduce-examples

kristinawlai committed:
fix merge and reconcile with main

Merge branch 'main' of https://github.com/UCD-SERG/serocalculator into reduce-examples

# Conflicts:
#	DESCRIPTION

d-morrison reviewed (approved):
Looks great!
